packages/components/src/components/hds/tabs/index.ts

```diff
@@ -42,15 +64,25 @@ export default class HdsTabsIndexComponent extends Component {
     return size;
   }
 
-  constructor() {
-    super(...arguments);
+  constructor(
+    owner: HdsTabsIndexComponent,
+    args: HdsTabsIndexSignature['Args']
+  ) {
+    super(owner, args);
 
     // this is to determine if the "selected" tab logic is controlled in the consumers' code or is maintained as an internal state
     this.isControlled = this.args.selectedTabIndex !== undefined;
   }
 
   get selectedTabIndex() {
     if (this.isControlled) {
+      assert(
```

Comment:
You're right. The SIZES code was a bad copy/paste from another assertion.

You're also correct that the assertion will never be false. Unfortunately, typescript is having a hard time discerning that fact and insists that `this.args.selectedTabIndex` can be `undefined`, even though this is impossible.

I've changed it to this instead of adding an assertion

`return this.args.selectedTabIndex!;`